examples/plot_minimal_pydeseq2_pipeline.py

```diff
@@ -95,51 +135,224 @@
 )
 
 # %%
-# Then, run DESeq2 on it
+# A :class:`DeseqDataSet` has two mandatory
+# arguments: a ``counts`` and a ``clinical`` dataframe, like the ones we've loaded in the
```

Comment:
I may be wrong but I don't think there should be the plural here since "counts" and "clinical" are acting as adjectives.
E.g. as in the [following example](https://www.numerade.com/ask/question/a-blue-and-a-green-billiard-ball-each-with-a-mass-of-015-kg-collide-directly-before-the-collision-the-blue-ball-had-a-speed-of-3-ms-while-the-green-ball-had-a-speed-of-2-ms-after-the-collision-the-gre/) you'd say "A blue and a green billiard ball" but not "A blue and a green billiard ball*s*"